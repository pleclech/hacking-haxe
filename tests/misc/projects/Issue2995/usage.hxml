--display Main.hx@92@usage