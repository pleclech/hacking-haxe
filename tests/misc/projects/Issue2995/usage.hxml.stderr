<list>
<pos>$$normPath(::cwd::/Main.hx):5: characters 16-26</pos>
<pos>$$normPath(::cwd::/Main.hx):9: characters 8-18</pos>
</list>
